en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
+The search can be case-sensitive or not (checkbox 'Case sensitive').
+The search can either be done as a plain-text or a regular-expression search (checkbox 'Regular expression').
 
-The logical complement to a refining group is a group that **includes its subgroups**, i.e. when selected, not only the group's own entries, but also its subgroups' entries are shown. In the groups tree, this type of group has a special icon \(this can be turned off in the preferences\).
+Obviously, this will work only for entries including the specified grouping field, and the quality of the grouping will depend on the content accuracy.
 
-In other words, this defines which entries are displayed when this group is selected:
+**%%%Help needed: True in 5.2? I do not understand the paragraph... So, if true in 5.2 and unclear to you please reword %%%**
+In the former case, JabRef allows to manually assign/remove entries to/from the group by simply appending/removing the search term to/from the content of the grouping field.
+This makes sense only for the `keywords` field or for self-defined fields, but obviously not for fields like `author` or `year`.
 
-* . Displays only this group's entries.
-* **intersection**. Displays entries contained in both this group and its supergroup.
-* **union**. Displays entries contained in this group or its subgroups.
+The content of the group is updated dynamically whenever the database changes.
 
-## Viewing a group's entries, combining multiple groups
+#### Using a free-form search expression 
 
-Selecting a group shows the entries contained in that group by highlighting them and, depending on the settings \(accessible by clicking the **Settings** button\), moving them to the top of the list and/or selecting them. These options are identical to those available for the regular search.
+This is similar to the above, but rather than search for a single search term on a single field, a [search expression syntax](search.md#advanced) can be used.
+It supports logical operators (`AND`, `OR`, `NOT`) and allows searching multiple fields.
 
-When multiple groups are selected, either the union or the intersection of their content is shown, depending on the current settings. This allows to quickly combine multiple conditions, e.g. if you have a static group `Extremely Important` to which you assign all extremely important entries, you can view the extremely important entries in any other group by selecting both groups \(this requires to have **Intersection** selected in the settings\).
+**%%% screenshot here: grab of the group type %%%**
+
+For example, the search expression `keywords=regression and not keywords=linear` groups entries concerned with non-linear regression.
+
+The content of the group is updated dynamically whenever the database changes.
```

Comment:
This sentence applies to all groups.

Reply:
I prefer to keep the sentence for each cases because I guess users do not read the help from the beginning to the end. I suppose they are likely to go to the kind of group they are interested in, and so I choose to repeat the sentence so that they find the information directly.